### PR TITLE
Nerfs sticky resin

### PR DESCRIPTION
**Pull request by Mister-moon1**
# About the pull request
Adds a build cap to sticky resin

# Explain why it's good for the game
Sticky resin right now is pretty awful ingame, not because it's bad but because it promotes the act of hivelords spending 20 minutes placing sticky resin on every single tile in caves, it should be used more strategically and develop where the combat is, rather than coating every square inch in 5 layers of it. 

Overall it will provide a more pleasant time for marines, increase the usage of hive clusters for the slowdown of hiveweeds and it will encourage builders to play more strategically

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
balance: Added a build cap to sticky resin of 25
/:cl:

### CODE REVIEW

**Comment by Diegoflores31** [comment]
my brother in christ you capped the wrong resin

File: code/modules/mob/living/carbon/xenomorph/resin_constructions.dm [hunk 1]
```diff
@@ -249,6 +249,7 @@ GLOBAL_VAR_INIT(resin_lz_allowed, FALSE)
 	construction_name = "fast resin"
 	cost = XENO_RESIN_FAST_COST
 	build_time = 1 SECONDS
+	max_per_xeno = 25
```
Comment (Diegoflores31) [inline code/modules/mob/living/carbon/xenomorph/resin_constructions.dm:252]: this is fast resin not sticky resin

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
you did this on the wrong type

**Comment by Cthulhu80** [comment]
bless ur heart young man

**Pull request closed without merging**